samples/VanillaJSTestApp/index_authCode.html

```diff
@@ -24,6 +24,7 @@
     const msalConfig = {
         auth: {
             clientId: "Enter_your_client_id",
+            clientSecret: "Enter_your_client_secret",
```

Comment:
Again, make it clear (with comments) that this is only needed for pre-release versions.

Reply:
Done, changed the name of the variable, added logger warnings, added comments in the readme.